code/__DEFINES/dcs/signals/signals_weather.dm

```diff
@@ -0,0 +1,5 @@
+// Signals related to weather and weather transitions
+// Sent primarily from weather_fsm.dm
+
+#define COMSIG_GLOB_Z_WEATHER_CHANGE "!z_weather_change"
+#define COMSIG_GLOB_Z_WEATHER_BROADCAST "!z_weather_broadcast"
```

Comment:
DMDoc, also why does this start with an exclamation mark? That makes it look like it's saying it's not what the text says

Reply:
![image](https://github.com/user-attachments/assets/b231a54d-6571-483a-977c-7749b03881ee)